pages/services/cassandra/2.6.0-3.11.4/security/index.md

```diff
@@ -0,0 +1,52 @@
+---
+layout: layout.pug
+navigationTitle: Security
+excerpt: Securing your service
+title: DC/OS Cassandra security
+menuWeight: 50
+model: /services/cassandra/data.yml
+render: mustache
+enterprise: true
+---
+
+# DC/OS {{ model.techName }} Security
+
+- The DC/OS {{ model.techName }} service allows you to create a service account to configure access for {{ model.techName }}. The service allows you to create and assign permissions as required for access.  
+
+- The DC/OS {{ model.techName }} service supports {{ model.techName }}'s native transport encryption mechanisms. The service provides automation and orchestration to simplify the usage of these important features.
```

Comment:
```suggestion
- The DC/OS {{ model.techName }} service supports {{ model.techName }}'s native transport encryption mechanisms. The service provides automation and orchestration to simplify the usage of the important features.
```

Reply:
these is mentioned because it refer to below mentioned features, i am not sure it can be changed to "the".

Reply:
Perhaps change "the important features" to "the following features".